Notepad.Extensions.Logging/WindowFinder.cs

```diff
@@ -12,11 +12,14 @@ public WindowFinder()
 
         readonly ObjectPool<WindowEnumerationState> statePool;
 
+        public string WindowName { get; internal set; }
+
         public WindowInfo FindNotepadWindow()
         {
             var stateObject = statePool.Get();
             try
             {
+                stateObject.WindowName = WindowName;
```

Comment:
Can we make this a method parameter of `FindNotepadWindow` instead of a public property of `WindowFinder`?

Reply:
If will force us to pass it into `NotepadLogger`. I think if logger get instance of `IWindowFinder` it should not be able to change window name.

If you think that logger should manage window finder behaviour I will change it.

Reply:
I don't see a problem with `NotepadLogger` reading the window name from the options and passing it to the finder. Ultimately the logger needs to find the window and write to it...